static/src/stylesheets/module/_weather.scss

```diff
@@ -27,13 +26,159 @@ $weather-large-size: 60px;
+        @include box-sizing(border-box);
+        width: gs-span(1) + $gs-gutter;
+        float: left;
+        padding-right: $weather-small-size + $gs-gutter / 4;
+        margin-right: $gs-gutter / 4;
+        text-align: right;
+
+        &:after {
+            content: '';
+            position: absolute;
+            top: 3px;
+            right: 0;
+            height: $gs-baseline + 1px;
+            border-right: 1px dotted colour(neutral-3);
+        }
+
+        .weather__icon {
+            position: absolute;
+            top: 0;
+            right: $gs-gutter / 4;
+        }
+    }
+
+    .has-page-skin & {
+        @include mq(wide) {
+            @include box-sizing(border-box);
+            padding-right: $weather-small-size + $gs-gutter / 4;
+            width: gs-span(1) + $gs-gutter;
+            float: left;
+            text-align: right;
+
+            .weather__icon {
+                position: absolute;
+                top: 0;
+            }
+
+            &:after {
+                content: '';
+                position: absolute;
+                top: 3px;
+                right: 0;
+                height: $gs-baseline + 1px;
+                border-right: 1px dotted colour(neutral-3);
+            }
+
+            .weather__icon {
+                right: $gs-gutter / 4;
+            }
+        }
+    }
+}
+
+.weather__desc {
+    @include fs-headline(2);
+    margin-right: 5px;
+
+    @include mq(leftCol) {
+        position: absolute;
+        bottom: 2px;
+    }
+
+    .has-page-skin & {
+        @include mq(wide) {
+            position: inherit;
+        }
+    }
+}
+
+.weather__time {
+    display: none;
+    color: colour(neutral-2);
+    @include fs-textSans(1);
+
+    @include mq(leftCol) {
+        display: block;
+    }
+
+    .has-page-skin & {
+        @include mq(wide) {
+            display: none;
+        }
+    }
+}
+
+.i.weather__icon {
+    width: $weather-small-size !important;
+    height: $weather-small-size !important;
+    margin: 0;
+    margin-top: -($gs-baseline / 3);
+
+    @include mq(leftCol) {
+        width: $weather-large-size !important;
+        height: $weather-large-size !important;
+        display: block;
+        margin: 0 auto ($gs-baseline / 4) ($gs-gutter * 2);
+    }
+
+    .has-page-skin & {
+        @include mq(wide) {
+            width: $weather-small-size !important;
+            height: $weather-small-size !important;
+            margin: 0;
+            margin-top: -($gs-baseline / 3);
+            display: inline-block;
+        }
+    }
+}
+
+.weather__toggle-forecast {
+    position: relative;
+    float: left;
+
+    @include mq(tablet) {
+        padding-right: $gs-gutter / 4;
+        margin-right: $gs-gutter / 4;
+
+        &:after {
+            content: '';
+            top: 3px;
+            right: 0;
+            position: absolute;
+            height: $gs-baseline + 1px;
+            border-right: 1px dotted colour(neutral-3);
+        }
```

Comment:
Although breakpoints would make that tricky. I think a mixin makes the most sense.